### PR TITLE
Use native ibexa api for filter

### PR DESCRIPTION
**Pull request by wizhippo**

### CODE REVIEW

File: bundle/Templating/Twig/BetterIbexaAdminUIRuntime.php [hunk 1]
```diff
@@ -4,26 +4,26 @@
...
+use Ibexa\Contracts\Core\Repository\ContentService;
 use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
-use Netgen\IbexaSiteApi\API\FilterService;
+use Ibexa\Contracts\Core\Repository\Values\Filter\Filter;
 use Twig\Extension\RuntimeExtensionInterface;
 
 final class BetterIbexaAdminUIRuntime implements RuntimeExtensionInterface
 {
-    private FilterService $filterService;
+    private ContentService $filterService;
```
Comment (emodric) [inline bundle/Templating/Twig/BetterIbexaAdminUIRuntime.php]: The property should also be renamed to `$contentService`

**Comment by emodric** [comment]
Thanks @wizhippo ! I have left a minor remark, but otherwise :+1:

**Review by emodric — APPROVED** [review]

File: bundle/Templating/Twig/BetterIbexaAdminUIRuntime.php (hunk 1 above)
Comment (wizhippo) [reply bundle/Templating/Twig/BetterIbexaAdminUIRuntime.php]: That's embarrassing.

**Review by pspanja — APPROVED** [review]

**Comment by pspanja** [comment]
Thanks @wizhippo! Released in [1.0.5](https://github.com/netgen/better-ibexa-admin-ui/releases/tag/1.0.5).

**Comment by wizhippo** [comment]
Thank you!  I love seeing the features we had in the old ez publish come back.

**Pull request merged**